docker/production/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '3'
+
+services:
+  edpn-frontend-docker-production:
```

Comment:
You'll probably want to consider healthchecks, maybe an external env file, see: https://github.com/ed-pilots-network/backend/blob/35-modulith-commodity-finder/docker-compose.yml for an example from backend.

That said, that's a production concern for later down the line, and simply to help ops out. We healthchecks, the Docker daemon can restart the service if it goes down (or some other orchestrator can get involved).

Likely not needed at this stage of the project.